doc/administrator/web/index.md

```diff
@@ -98,6 +98,21 @@ The main configuration parameters you will need to change are:
   * `ncbi.upload.namespace` - Prefix for file upload identifiers to NCBI. The namespace is used to guarantee upload IDs are unique.  This configuration option is used as a placeholder and may still be set by the user.
 5. **Security configuration**
  * `security.password.expiry` - The number of days a password is valid for in IRIDA.  After a password expires the user will be required to create a new one.  Passwords cannot be reused.
+6. **OAuth2 JWK security configuration** - IRIDA uses JWT (Json Web Tokens) for OAuth2 and as such requires a JWK (Json Web Key) in RSA format to encrypt and decrypt these tokens.
+ * `oauth2.jwk.rsakey.id=CHANGEME` - The id for the RSA private and public keys (used to select the correct keys for encryption and decryption)
```

Comment:
I think leaving it as `CHANGEME` wouldn't be a security risk. This key is used internally by IRIDA to determine which public/private keys to encrypt and decrypt the access tokens. If someone were to change the ID and restart their server old access tokens would no longer work as IRIDA would not be able to find the correct keys to use.

Reply:
Okay, that makes sense. I figured that was the case but wanted to check.

Reply:
Actually, thinking this over a bit, I wonder if it could be renamed from **CHANGEME** to something like **key_id** or some other default string?

My reason is because I think of phrases like `CHANGEME` as implying it is a requirement to change, when it is really not a requirement, but only optional.

Reply:
Fixed in [bc31bfd](https://github.com/phac-nml/irida/pull/1339/commits/bc31bfd2da4bd29e9dd7ff503b9921c866ff6381)